### PR TITLE
Add Nimbus helm chart

### PR DESCRIPTION
**Pull request by chingyi-lin**

### CODE REVIEW

File: nimbus/helm/templates/nimbus-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{{- range $key, $nimbusClient := .Values.nimbus.clients }}
```
Comment (LaContra) [inline nimbus/helm/templates/nimbus-deployment.yaml]: If the array is named nimbus.clients, the variable can be $client with the nimbus to make it consistent.

From our previous discussion, it's also fine to keep it as it is if you think nimbusClient makes more sense semantically.

File: nimbus/helm/templates/nimbus-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $nimbusClient.name }}
+  namespace: nimbus
```
Comment (LaContra) [inline nimbus/helm/templates/nimbus-deployment.yaml]: For other clients, namespace is also a variable in the helm chart. Are we going to change the behavior?

File: nimbus/helm/templates/nimbus-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        - --non-interactive
+        - --tcp-port={{ $nimbusClient.p2pTcpPort }}
+        - --udp-port={{ $nimbusClient.p2pUdpPort }}
+        - --data-dir=/data/nimbus/beacon
+        - --validators-dir=/data/nimbus/validators
+        - --secrets-dir=/data/nimbus/secrets
+        - --web3-url={{ $nimbusClient.web3Provider }}
+        {{- range .fallbackWeb3Providers }}
+        - --web3-url={{ . }}
+        {{- end }}
```
Comment (LaContra) [inline nimbus/helm/templates/nimbus-deployment.yaml]: I can see the benefits of this approach is that web3Provider is required field and fallbackWeb3Providers is optional.

The other option is that we turn web3Provider into web3Providers and the variable will take an array instead. The good part about this approach is that we only need to iterate through web3Providers. 

Just want to point out there's another way. I'm fine with either approach.

File: nimbus/helm/values.yaml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+# A required section that contains Nimbus client information.
+nimbus:
+  # This field determines how long it will wait before the Nimbus client starts or restarts.
+  # It's a way to protect the validator client from double attesting or block proposing when the DB is out-of-date.
+  startWaitTime: 780
+  # A required section that contains information for all Nimbus clients
+  clients:
+    # This section contains information for one Nimbus client.
+    # "nimbusClient1" is the name of this Nimbus client section. You can change it to other strings as long as it's different than other section names.
```
Comment (LaContra) [inline nimbus/helm/values.yaml]:
```suggestion
    # "client1" is the name of this Nimbus client section. You can change it to other strings as long as it's different than other section names.
```

File: nimbus/helm/templates/nimbus-deployment.yaml (hunk 3 above)
Comment (chingyi-lin) [reply nimbus/helm/templates/nimbus-deployment.yaml]: right I prefer to separate them into two different values for now

File: nimbus/helm/templates/nimbus-deployment.yaml (hunk 1 above)
Comment (chingyi-lin) [reply nimbus/helm/templates/nimbus-deployment.yaml]: changed it to `$client`

**Review by LaContra — APPROVED** [review]

**Pull request merged**